packages/docs/developer-resources/integrations/checklist.md

```diff
@@ -20,7 +20,9 @@ Celo has [WIP QR code standard CIP16](https://github.com/celo-org/celo-proposals
 Please read more under [Custody](./custody.md), but here is a shortened version:
 
 ### Detect Transfers
-Stable-value currencies, currently just cUSD, is a contract `StableToken` that can be accessed via the ERC20 interface. The native asset CELO can be accessed via the `GoldToken` ERC20 interface, or natively, similar to ETH on Ethereum.
+Stable-value currencies, currently cUSD and cEUR, are contracts, `StableToken` and `StableToken` respectively, that can be accessed via the ERC20 interface.  The native asset CELO can be accessed via the `GoldToken` ERC20 interface, or natively, similar to ETH on Ethereum.
```

Comment:
should we say instead:
Stable-value currencies, cUSD and cEUR, are `StableToken` contracts that can be accessed via the ERC20 interface.

Reply:
Thanks!